jsk_unitree_robot/cross/ros1_dependencies_build_scripts/1044-libfcl-dev

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+set -xeu -o pipefail
+
+DEBIAN_DIR=/home/user/ros1_dependencies_sources/src/libfcl-dev/debian/debian
+SOURCE_DIR=/home/user/ros1_dependencies_sources/src/libfcl-dev/fcl-0.5.0
+
+cd ${DEBIAN_DIR}/patches
+for patch_file in $(grep -v ^# series); do
+    OUT="$(patch -p1 --forward --directory ${SOURCE_DIR} < ${patch_file} | tee /dev/tty)" || echo "${OUT}" | grep "Skipping patch" -q || (echo "$OUT" && false) || echo "OK"
+done
+
+mkdir -p /home/user/ros1_dependencies_sources/build/libfcl-dev
+cd /home/user/ros1_dependencies_sources/build/libfcl-dev
+LD_LIBRARY_PATH=/opt/jsk/${INSTALL_ROOT}/ros1_dependencies/lib:${LD_LIBRARY_PATH}
```

Comment:
it did not find `libccd` , when building test code, so I just skip it. I haven't tested if fcl works in running environmnet (in a real robot) 
https://github.com/jsk-ros-pkg/jsk_robot/blob/ae9ff8e3a3ad84256336a3e91e68d5f7dc88231a/jsk_unitree_robot/cross/ros1_dependencies_build_scripts/1044-libfcl-dev#L18